deploy/cr.yaml

```diff
@@ -44,7 +47,7 @@ spec:
       persistentVolumeClaim:
         resources:
           requests:
-            storage: 2G
+            storage: 20G
```

Comment:
Why do we need 20G? Can we use 6G as in PXCC operator?

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/164/commits/f1c37b264d833ef3d9ba3f299d7051f721825f97